language/unit_tests/test_client.py

```diff
@@ -17,28 +17,33 @@
 
 class TestClient(unittest.TestCase):
 
-    def _getTargetClass(self):
+    @staticmethod
+    def _get_target_class():
         from google.cloud.language.client import Client
         return Client
 
-    def _makeOne(self, *args, **kw):
-        return self._getTargetClass()(*args, **kw)
+    def _make_one(self, *args, **kw):
+        return self._get_target_class()(*args, **kw)
 
     def test_ctor(self):
+        import mock
         from google.cloud.language.connection import Connection
 
-        creds = _Credentials()
+        creds = mock.Mock()
+        creds.create_scoped_required.return_value = False
         http = object()
-        client = self._makeOne(credentials=creds, http=http)
+        client = self._make_one(credentials=creds, http=http)
         self.assertIsInstance(client.connection, Connection)
         self.assertIs(client.connection.credentials, creds)
         self.assertIs(client.connection.http, http)
 
     def test_document_from_text_factory(self):
+        import mock
         from google.cloud.language.document import Document
 
-        creds = _Credentials()
-        client = self._makeOne(credentials=creds, http=object())
+        creds = mock.Mock()
+        creds.create_scoped_required.return_value = False
```

Comment:
Not a win over our `_Credentials` class IMO:  it is noisier and less comprehensible.  Also, without setting a `spec` on the mock, it is willing to return attrs promiscuously, which is a Bad Thing.